### PR TITLE
adding getSocialScores

### PR DESCRIPTION
**Pull request by lawzlo**
@stkem @leogrim @yingjieMiao

### CODE REVIEW

File: kifi-backend/modules/curator/app/com/keepit/curator/commanders/UriScoringHelper.scala [hunk 1]
```diff
@@ -42,8 +53,31 @@ class UriScoringHelper @Inject() (cortex: CortexServiceClient) {
...
+      //convert user scores seq to map, assume there is no duplicate userId from graph service
+      graph.getConnectedUserScores(items.head.userId, avoidFirstDegreeConnections = false).map { socialScores =>
+        val socialScoreMap = socialScores.map { socialScore =>
+          (socialScore.userId, socialScore.score.toFloat)
+        }.toMap
+
+        items.map(item =>
+          item.keepers match {
+            case Keepers.TooMany => 0.0f
+            case Keepers.ReasonableNumber(_) => {
```
Comment (stkem) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/UriScoringHelper.scala:71]: The content of `Keepers.ReasonableNumber` are the people who kept this uri. There is no need to look them up in the database again.

File: tools/etc/init.d/showbox [hunk 2]
```diff
@@ -4,35 +4,23 @@
 # description: Java deamon script
```
Comment (stkem) [inline tools/etc/init.d/showbox:4]: what happened here?

File: kifi-backend/modules/curator/app/com/keepit/curator/commanders/UriScoringHelper.scala (hunk 1 above)
Comment (lawzlo) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/UriScoringHelper.scala:71]: Oh. Cool. Fixing

File: tools/etc/init.d/showbox (hunk 2 above)
Comment (yingjieMiao) [inline tools/etc/init.d/showbox:4]: rename typo

Comment (lawzlo) [inline tools/etc/init.d/showbox:4]: restored shoebox config file

**Pull request closed without merging**